composer.json

```diff
@@ -41,7 +41,6 @@
     },
     "suggest": {
         "ext-mbstring": "For handling non-UTF8 characters in cache signature.",
-        "ext-xml": "For better performance.",
```

Comment:
in whole project we go with approach to be future-ready instead of legacy-ready.
if keep sth, i would rather keep `suggest: php7.2` :D but that's pointless, so I removed it